.vscode/launch.json

```diff
@@ -46,14 +46,16 @@
       "webRoot": "${workspaceFolder}"
     },
     {
-      "name": "Debug Jest test",
+      "name": "Debug UI test",
       "type": "node",
       "request": "launch",
       "runtimeExecutable": "yarn",
       "runtimeArgs": ["run", "jest", "--runInBand", "${file}"],
       "console": "integratedTerminal",
       "internalConsoleOptions": "neverOpen",
-      "port": 9229
```

Comment:
Apparently if you're specifying `"request": "launch"` then you don't need a `port`. My editor was warning me about this so I removed it at the same time